### ISSUE TITLE
Resolving issues does not update beta kata approval status

### ISSUE THREAD

**Issue by Voileexperiments** [codewars/codewars.com#1088]
A follow-up to #1037 .

I was experimenting with [this kata](https://www.codewars.com/kata/function-coercion-errors) today:

- Kata is awaiting issues to be resolved
- I solved the kata and gave vote/ranking feedback
- Then I resolved the issues on this kata. However, after resolving the issues, the kata still shows "awaiting issues to be resolved"
- I retracted my vote to see if it refreshes the beta kata status. Doing this makes the kata approvable
- Then I re-applied my feedback again, and the kata stays in "ready to be approved status"

tl;dr The above should make it pretty clear that resolving an issue doesn't update a beta kata's approval status, which is incorrect because resolving an issue *can* affect beta kata approval status.

**Comment by kazk** [comment]
The fix for #2041 should've also fixed this. Please let me know if this is still happening.
